tests/Etc/tmp/queuetest.json

```diff
@@ -0,0 +1 @@
+{"tenant_id":"The current tenant id is: acme"}
```

Comment:
Can you:
1. delete this file, and commit that change (don't push)
2. add the file to `.gitignore`, commit
3. push
4. make sure the file is not shown on GH

Reply:
It's also possible that I already added it to `.gitignore` and this file was created before I did that. So removing it might not have an effect. So you might need to do this before the steps I outlined above:
- delete the file from `.gitignore`, commit
- then the steps that I wrote above

Reply:
Did this and the file is not here anymore 👍 resolving this